kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/NamespaceQuotaExample.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.kubernetes.examples;
+
+import io.fabric8.kubernetes.api.model.Namespace;
+import io.fabric8.kubernetes.api.model.NamespaceBuilder;
+import io.fabric8.kubernetes.api.model.Quantity;
+import io.fabric8.kubernetes.api.model.ResourceQuota;
+import io.fabric8.kubernetes.api.model.ResourceQuotaBuilder;
+import io.fabric8.kubernetes.client.APIGroupNotAvailableException;
+import io.fabric8.kubernetes.client.Config;
+import io.fabric8.kubernetes.client.ConfigBuilder;
+import io.fabric8.kubernetes.client.DefaultKubernetesClient;
+import io.fabric8.kubernetes.client.KubernetesClient;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class NamespaceQuotaExample {
+  private static final Logger logger = LoggerFactory.getLogger(NamespaceQuotaExample.class);
+
+  public static void main(String[] args) throws InterruptedException {
+    String master = "https://localhost:8443";
+
+    if (args.length == 1) {
+      master = args[0];
+    }
+
+    Config config = new ConfigBuilder().withMasterUrl(master).build();
+    try (final KubernetesClient client = new DefaultKubernetesClient(config)) {
+      try  {
+        // Creating namespace
+        Namespace ns = new NamespaceBuilder().withNewMetadata().withName("namespacetest").addToLabels("hello", "world").endMetadata().build();
```

Comment:
rather than hardcoding the string value `namespacetest` , you could have created a Constant string @kunal-kushwaha

Reply:
Fixed